src/main/drivers/exti.h

```diff
@@ -25,9 +25,9 @@
 #include "drivers/io_types.h"
 
 typedef enum {
-    EXTI_TRIGGER_RISING = 0,
-    EXTI_TRIGGER_FALLING = 1,
-    EXTI_TRIGGER_BOTH = 2
+    UEXTI_TRIGGER_RISING = 0,
```

Comment:
What does `UEXTI` stand for? (Not that I currently have a suggestion for a better name.)

Reply:
Universal EXTI...

Reply:
Doesn't really congeal with me - other people looking at it won't understand / know how to use.
Maybe `BETAFLIGHT_EXTI...`?

Reply:
Lemme do that.